agr_mvs/Turno.py

```diff
@@ -0,0 +1,185 @@
+
+    def getS(self):
+        return self.S
+    
+    def getp(self):
+        return self.p
+
+      # Método para calcular el µ
+    '''
+    Para el cálculo del µ, se ha decidido que el valor de µ será el máximo de clientes que se han atendido en una hora.
+    Se va a dividir entre 60 para trabajar con minutos en los posteriores cálculos.
+    '''
+    def calculo_µ(self):
+        max_afluencia = 0
+        afluencia = 0
+        if (self.Tipo_turno == TipoTurno.ALMUERZO):
+            hora_actual = 13
+            hora_fin = 18
+            while (hora_actual <= hora_fin):
+                for ticket in self.Tickets:
+                    hora = ticket.fecha_ticket.hour
+                    if (hora == hora_actual):
+                        afluencia += 1
+                if (afluencia > max_afluencia):
+                    max_afluencia = afluencia
+                hora_actual += 1
+                afluencia = 0
+            self.µ = (max_afluencia/60.0)
+        else:
+            hora_actual = 18
+            hora_fin = 23
+            while (hora_actual <= hora_fin):
+                for ticket in self.Tickets:
+                    hora = ticket.fecha_ticket.hour
+                    if (hora == hora_actual):
+                        afluencia += 1
+                if (afluencia > max_afluencia):
+                    max_afluencia = afluencia
+                hora_actual += 1
+                afluencia = 0
+            self.µ = (max_afluencia/60.0)
+
+    '''
+    Para el calculo de λ debemos de tener en cuenta que lo hacemos por hora,
+    con fin de simplificar el cálculo, vamos a realizar una división entre el numero de tickets 
+    que tenemos en el turno y las horas del turno. (Representamos cada cliente por un ticket sin valorare el importe del ticket)
+    El resultado lo vamos a dividir entre 60 para trabajar con minutos en los posteriores cálculos.
+    '''
+    def calculo_λ(self):
+        self.λ = (len(self.Tickets)/HORAS_TURNO)/60.0
+    
+    '''
+    Calculo variables elementales para el calculo de las variables estadísticas
+    '''
+    def calculo_variables_elementales(self):
+        if (self.µ == None or self.λ == None):
+            self.calculo_µ()
+            self.calculo_λ()
+    
+    '''
+    Mediante esta funcion vamos a poder calcular la saturacion de la tienda.
+    '''
+    def calculo_p(self):
+        self.calculo_variables_elementales()
+        p = self.λ/(self.µ*self.S)
+        return p
+    
+    
+    '''
+    Calculo del numero promedio de clientes en la cola: Lq = p / Ls (Siendo Ls el numero promedio de clientes que entran al sistema)
+    Ls = λ / (µ - λ)
+    '''
+    def calculo_L(self):
+        p = self.calculo_p()
+        l = p * (self.λ / (self.µ - self.λ))
+        return l
+
+    '''
+    Calculo del tiempo promedio de espera de un cliente en la cola Wq = Lq/λ ( Vamos a expresar λ en minutos para que sea mas orientativo el resultado) 
```

Comment:
Igual, dale nombres significativos.

Reply:
Igual que le he comentado anteriormente he cambiado todo excepto las variables elementales ya que están suficientemente explicadas y son mas legibles por su letra que por lo que aportan

Reply:
Puedes llamar a las variables como te dé la gana, siempre que sean nombres significativos. El interfaz o API de una clase no tiene por qué tener el mismo nombre. No sé de dónde os habéis sacado eso, igual que los célebres getters y setters que metéis por todos lados.